source/includes/rest/_subscribers.md

```diff
@@ -16,6 +16,8 @@
   "zip": "90210",
   "country": "US",
   "phone": "555-555-5555",
+  "sms_phone_number": "+16125551212",
```

Comment:
@jacobpmeyer is the phone number exposed like in this payload??

Reply:
I thought that was the case, but I don't think we ever discussed it.